scripts/newsletter.js

```diff
@@ -194,4 +190,6 @@ const NewsletterForm = {
     }
 };
 
+NewsletterForm.init();
+
 export default NewsletterForm;
```

Comment:
**Nit:** Do we still need this line if we're not exporting the file into another file?

Reply:
oooo, smort